examples/scipy/README.md

```diff
@@ -0,0 +1,57 @@
+# [SciPy](https://scipy.org/) sparse matrices in ZenML
+
+An elementary data structure widely used in `sklearn` to store sparse data more efficiently.
+
+
+## 🧰 How the example is implemented
+
+We train a simple linear model to classify text based on sentiment.
```

Comment:
As you have included that dataset as a csv file it would be good (and maybe even required by its licence) to include areference to the dataset here.